packages/fxa-auth-server/scripts/delete-inactive-accounts/enqueue-inactive-account-deletions.ts

```diff
@@ -543,7 +543,7 @@ const init = async () => {
           // Note that the fxa cloud tasks lib already emitted some statsd metrics
           statsd.increment(
             'cloud-tasks.inactive-account-email.enqueue.error-code',
-            [cloudTaskQueueError.code as unknown as string]
+            { errorCode: cloudTaskQueueError.code as unknown as string }
```

Comment:
While the types in (and the docs for) hot-shots allow an array here Telegraf appears to be not to pleased about it.